automation/docker/Dockerfile

```diff
@@ -85,13 +81,9 @@ RUN ./gradlew clean \
     && ./gradlew androidDependencies \
     && ./gradlew --stop \
     && ./gradlew --no-daemon assemble \
-    && ./gradlew --no-daemon test \
+    && ./gradlew --no-daemon -Pcoverage test \
```

Comment:
If the point of the `./gradle` commands here are to download dependencies (so that they'll be cached in the image), I'm not sure that we need to do `-Pcoverage` (it doesn't influence the downloaded dependencies, does it?)

Reply:
Without `-Pcoverage` we didn't download the dependencies needed for generating the coverage reports. That's primarily jacoco and its transitive dependencies. I recently updated the Docker image and then a task failed because it couldn't download jacoco and then I realized that we never do that when building the image. :)